### ISSUE TITLE
[simprod-scripts] polyplopia with ppc (Trac #2051)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2043]
Running polyplopia with the ppc option:

```text
FATAL (MPHitFilter): No I3MCPESeriesMap named BackgroundI3MCPESeriesMap found in frame (MPHitFilter.cxx:73 in virtual void MPHitFilter::DAQ(I3FramePtr))
ERROR (I3Module): hitfilter: Exception thrown (I3Module.cxx:118 in void I3Module::Do(void (I3Module::*)()))
ERROR (I3Module): ppc: Exception thrown (I3Module.cxx:118 in void I3Module::Do(void (I3Module::*)()))
Traceback (most recent call last):
File "/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib/python2.7/runpy.py", line 162, in _run_module_as_main
"__main__", fname, loader, pkg_name)
File "/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib/python2.7/runpy.py", line 72, in _run_code
exec code in run_globals
File "/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py", line 110, in <module>
main()
File "/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py", line 103, in main
run(**args)
File "/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py", line 81, in run
ret = instance.Execute(stats)
File "/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/metaprojects/simulation/V05-01-02/lib/icecube/simprod/modules/polyplopia.py", line 126, in Execute
tray.Execute()
File "/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/metaprojects/simulation/V05-01-02/lib/I3Tray.py", line 256, in Execute
super(I3Tray, self).Execute()
RuntimeError: No I3MCPESeriesMap named BackgroundI3MCPESeriesMap found in frame (in virtual void MPHitFilter::DAQ(I3FramePtr))
```

Works fine when I use polyplopia with clsim.

Migrated from https://code.icecube.wisc.edu/ticket/2051
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "Running polyplopia with the ppc option:\n\n{{{\nFATAL (MPHitFilter): No I3MCPESeriesMap named BackgroundI3MCPESeriesMap found in frame (MPHitFilter.cxx:73 in virtual void MPHitFilter::DAQ(I3FramePtr))\nERROR (I3Module): hitfilter: Exception thrown (I3Module.cxx:118 in void I3Module::Do(void (I3Module::*)()))\nERROR (I3Module): ppc: Exception thrown (I3Module.cxx:118 in void I3Module::Do(void (I3Module::*)()))\nTraceback (most recent call last):\nFile \"/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib/python2.7/runpy.py\", line 162, in _run_module_as_main\n\"__main__\", fname, loader, pkg_name)\nFile \"/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib/python2.7/runpy.py\", line 72, in _run_code\nexec code in run_globals\nFile \"/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py\", line 110, in <module>\nmain()\nFile \"/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py\", line 103, in main\nrun(**args)\nFile \"/cvmfs/icecube.opensciencegrid.org/iceprod/master/RHEL_6_x86_64/lib/python2.7/site-packages/iceprod/core/exe_helper.py\", line 81, in run\nret = instance.Execute(stats)\nFile \"/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/metaprojects/simulation/V05-01-02/lib/icecube/simprod/modules/polyplopia.py\", line 126, in Execute\ntray.Execute()\nFile \"/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/metaprojects/simulation/V05-01-02/lib/I3Tray.py\", line 256, in Execute\nsuper(I3Tray, self).Execute()\nRuntimeError: No I3MCPESeriesMap named BackgroundI3MCPESeriesMap found in frame (in virtual void MPHitFilter::DAQ(I3FramePtr))\n}}}\n\nWorks fine when I use polyplopia with clsim.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo simulation", 
    "summary": "[simprod-scripts] polyplopia with ppc", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2017-07-07T16:28:24", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170707T16:54:00: juancarlos changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170919T14:51:33: 
* juancarlos commented:

> In changeset 158255:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="158255"
> Closes #2043 polyplopia with PPC
> ```


* juancarlos changed status from "accepted" to "closed"
* juancarlos changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas commented:

> Milestone July 2017 deleted


* olivas changed milestone from "July 2017" to ""
